JS/modal.js

```diff
@@ -0,0 +1,171 @@
+  document.getElementById('Contact').classList.add('blur');
+  document.getElementById('html').innerHTML = workCards[0].technologies[0];
+  document.getElementById('css').innerHTML = workCards[0].technologies[1];
+  document.getElementById('js').innerHTML = workCards[0].technologies[2];
+  document.getElementById('mobiletonic').src = workCards[0].image;
+  document.getElementById('site').href = workCards[0].linkLive;
+  document.getElementById('source').href = workCards[0].linkSource;
+};
+
+// document.getElementById('btn1').onclick = () => {
+//   document.getElementById('card-container').classList.add('star');
+//   document.getElementById('body').classList.add('blur');
+//   document.getElementById('Portfolio').classList.add('blur');
+//   document.getElementById('About').classList.add('blur');
+//   document.getElementById('Contact').classList.add('blur');
+//   document.getElementById('l1').innerHTML = workCards[1].technologies[0];
+//   document.getElementById('l2').innerHTML = workCards[1].technologies[1];
+//   document.getElementById('l3').innerHTML = workCards[1].technologies[2];
+//   document.getElementById('mobiletonic').src = workCards[1].image;
+//   document.getElementById('site').href = workCards[1].linkLive;
+//   document.getElementById('source').href = workCards[1].linkSource;
+// };
+
+// document.getElementById('btn2').onclick = () => {
+//   document.getElementById('card-container').classList.add('star');
+//   document.getElementById('body').classList.add('blur');
+//   document.getElementById('Portfolio').classList.add('blur');
+//   document.getElementById('About').classList.add('blur');
+//   document.getElementById('Contact').classList.add('blur');
+//   document.getElementById('l1').innerHTML = workCards[2].technologies[0];
+//   document.getElementById('l2').innerHTML = workCards[2].technologies[1];
+//   document.getElementById('l3').innerHTML = workCards[2].technologies[2];
+//   document.getElementById('mobiletonic').src = workCards[2].image;
+//   document.getElementById('site').href = workCards[2].linkLive;
+//   document.getElementById('source').href = workCards[2].linkSource;
+// };
+
+// document.getElementById('btn3').onclick = () => {
+//   document.getElementById('card-container').classList.add('star');
+//   document.getElementById('body').classList.add('blur');
+//   document.getElementById('Portfolio').classList.add('blur');
+//   document.getElementById('About').classList.add('blur');
+//   document.getElementById('Contact').classList.add('blur');
+//   document.getElementById('l1').innerHTML = workCards[1].technologies[0];
+//   document.getElementById('l2').innerHTML = workCards[1].technologies[1];
+//   document.getElementById('l3').innerHTML = workCards[1].technologies[2];
+//   document.getElementById('mobiletonic').src = workCards[3].image;
+//   document.getElementById('site').href = workCards[1].linkLive;
+//   document.getElementById('source').href = workCards[1].linkSource;
+// };
```

Comment:
- [ ] Dear @mertkantaroglu, Commented codes are against a good practice for clean code as this will make your code messy. Pleas remove commented code and remember you can always go back with the commit history to see a code you have used once.